modules/efk-logging-curator-scripted.adoc

```diff
@@ -39,18 +40,22 @@ $ oc create configmap curator \
     ; sleep 1
 ----
 +
-* Alternatively, if you are using the *action file*:
-+
-.. Copy the Curator configuration file and the `action` file
+The next iteration will use this configuration.
+
+* If you are using the *action* file:
+
+. Create and modify the configuration:
+
+.. Copy the Curator configuration file and the *action* file
 from the Curator configuration map and create separate files for each:
 +
 ----
 $ oc extract configmap/curator --keys=curator5.yaml,actions.yaml --to=/my/config
 ----
 +
-.. Edit the *_/my/config/curator5.yaml_* and *_/my/config/actions.yaml_* files.
+.. Edit the *_/my/config/curator5.yaml_* and *_/my/config/action.yaml_* files.
```

Comment:
When I wrote the scripts that generate the action file I was still calling it _"actions"_ file - when referring to the file as a path somewhere on a computer, it is still `actions.yaml`.